__tests__/__snapshots__/ordCdsrc.test.js.snap

```diff
@@ -105,6 +56,21 @@ exports[`Tests for default ORD document when .cdsrc.json is present Successfully
       "version": "1.0.0",
       "visibility": "public",
     },
+    {
```

Comment:
@zongqichen : This is "coming" from __tests__\bookshop\ord\custom.ord.json. Not sure if we want to inject this into the resulting ORD Document. What is your opinion?